src/meta-assistant/generic/time-utils.ts

```diff
@@ -0,0 +1,147 @@
+import { DailyCombinedData, Dates, TimeRegistrations, PreviousWorkdayDates, NonProjectTime, DisplayValues, CalculateWorkedTimeAndBillableHoursResponse } from "src/types/meta-assistant/index";
+import { DateTime, Duration } from "luxon";
+import { PersonTotalTime } from "src/generated/client/api";
+
+/**
+ * Namespace for time utilities
+ */
+namespace TimeUtilities {
+
+  /**
+   * Converts minutes into hours and minutes
+   *
+   * @param duration in minutes from timebank Data
+   * @returns string of timebank data converted to hours and minutes
+   */
+  export const timeConversion = (duration: number): string => {
+    const dur = Duration.fromObject({ minutes: duration });
+    const time = dur.shiftTo("hours", "minutes");
+    return `${time.hours}h ${time.minutes} minutes`;
```

Comment:
Why formatting is not done with luxon?

Reply:
No good reason, again could this be left as part of a future trainee task